### DISCUSSION TITLE
Require API for exporting tabular data into excel

### DISCUSSION THREAD

**Discussion by tribhuwanphulera1** [grafana/grafana#55209] [General]
If would want to integrate Grafana with another application and would wish to use it as report generating engine than API based Excel export is required to achieve the same.
Hope Grafana team will understand that and consider it as a genuine requirement.
Please support and implement!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
